cmd/updater/update.sh

```diff
@@ -330,20 +340,75 @@ function validate_update() {
     return 0
 }
 
+function has_system_service() {
+    local dd="$1"
+    local path
+
+    path=$(awk -F= '{ print $2 }' <(systemctl show -p FragmentPath "algorand@$(systemd-escape "$dd")"))
+    if [ "$path" != "" ]; then
+        return 0
+    fi
+    return 1
+}
+
+function has_user_service() {
+    local dd="$1"
+    local path
+
+    path=$(awk -F= '{ print $2 }' <(systemctl --user show -p FragmentPath "algorand@$(systemd-escape "$dd")"))
+    if [ "$path" != "" ]; then
+        return 0
+#        state=$(systemctl --user show -p ActiveState --value "algorand@$(systemd-escape "$dd")")
```

Comment:
This need to be dropped, right ?

Reply:
Yes, in fact I should combine those two functions because they're virtually identical.

Reply:
I removed the commented-out code and combined the two functions into one.